2.10/acd.in

```diff
@@ -31,20 +31,23 @@ community.vmware
 community.windows
 community.zabbix
 containers.podman
+cyberark.conjur
 cyberark.pas
-dellemc_networking.os10
-#dellemc_networking.os6
-#dellemc_networking.os9
+#dellemc.os10 # https://github.com/ansible-collections/dellemc.os10/issues/20
```

Comment:
Since dellemc_networking.os10 is working, why do we want to disable that right now?

Reply:
Was published under old name, not new https://galaxy.ansible.com/dellemc

Reply:
Ah, I may have missed understood this. 
I guess we could still use the old one. though we still need to track that the correctly named one doesn't exist. 
I'm happy to take your lead here.
Maybe we should keep it in, as otherwise we've regressed the next build of `ansible`